lib/ast/primitive.ts

```diff
@@ -971,16 +971,21 @@ Stream.Timer.prototype.isTimer = true;
 export class AtTimerStream extends Stream {
     time : Value[];
     expiration_date : Value|null;
+    once : Value|false;
 
     constructor(location : SourceRange|null,
                 time : Value[],
                 expiration_date : Value|null,
+                once : Value|false,
                 schema : FunctionDef|null) {
         super(location, schema);
 
         assert(Array.isArray(time));
         this.time = time;
 
+        assert(once instanceof Value);
```

Comment:
I'm surprised this assert wasn't a problem given you pass `false` in a bunch of places.